lib/handbook/addon/docs/components/tags-widget/editable/template.hbs

```diff
@@ -0,0 +1,19 @@
+{{#docs-demo as |demo|}}
+    {{#demo.example name='tags-widget.editable.hbs'}}
+        {{tags-widget
+            tags=tags
+            addTag=(action addTag)
+            removeTag=(action removeTag)
+            readOnly=readOnly
+            showAdd=true
+            analyticsScope='Handbook Tags'
+        }}
+
+    {{/demo.example}}
+
+    {{#bs-button class='center-block m-b-lg' onClick=(action (mut readOnly) (not readOnly))}}
+        set: readOnly = {{not readOnly}}
+    {{/bs-button}}
```

Comment:
Is this something that the user will normally be toggling? It seems like this will normally just depend on permissions. I think probably just adding another example illustrating readonly makes sense. This example will also show that `addTag` and `removeTag` are not required when `readOnly=true`.